ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -42,19 +42,24 @@ cabalFileLibraries =
     , "compiler/ghc.cabal"
     ]
 
+-- These predicates are used so ubiquitously it makes sense to
+-- introduce some sugar.
+isGhcMaster, isGhc808, isGhcGreaterEqual810 :: GhcFlavor -> Bool
+isGhcMaster ghcFlavor = ghcFlavor == GhcMaster
+isGhc808 ghcFlavor = ghcFlavor `notElem` [ GhcMaster, Ghc8101]
+isGhcGreaterEqual810 ghcFlavor = ghcFlavor `elem` [ GhcMaster, Ghc8101 ]
```

Comment:
Would it make sense to define an `Ord` instance?

Reply:
That's a great idea.

Reply:
Note that `Ord` is a bit tricky here. You’ll also want to override `Eq` and then you end up with `Da` and `Ghc881` being identical which could cause problems if you want to differentiate them via `Eq` (not sure we do). If you want to avoid that go for a custom comparison function.

Reply:
As we concluded off ticket, we'll hold off on this for now and remove the sugar.